src/components/Search/Button/EditRecord.js

```diff
@@ -24,11 +23,10 @@ const EditRecordButton = ({ ...props }) => {
     const { router, toggleFilterPane } = props;
     const id = findParam('id');
     toggleFilterPane();
-    // lockRecord(id);
+    lockRecord(id);
```

Comment:
This changes the behavior of the edit button. Why is it included in this PR?

Reply:
I would also say to leave it commented for the moment

Reply:
It might've been a lint thing. I'll undo it.